### DISCUSSION TITLE
Briefcase and git?

### DISCUSSION THREAD

**Discussion by martinandersen3d** [beeware/beeware#115] [Q&A]
👉 **What folder is it that I'm supposed to put in source controle ???** 👈

This is the first time I use breifcase and develop for linux/win/macos

Thanks 🙏

![image](https://user-images.githubusercontent.com/19984581/128610996-513fba53-7c4a-4a60-92b2-26ace9980abe.png)

**Comment by freakboy3742** [comment]
From your screenshot, the only files you'd put in version control are:

- `helloworld`
  - `src`
    - `helloworld`
      - `resources`
        - *all files*
      - `__init__.py`
      - `__main__.py`
      - `app.py`
  - .gitignore
  - LICENSE
  - pyproject.toml
  - README.rst
 
Everything else is an "output artefact" of the project.
 * `beeware-venv` is your virtual environment; that can be recreated at any time
 * `helloworld/.vscode` is the configuration of VSCode
 * `linux` (as well as `macOS`, `android` and any other platform directory) are build artefacts. They're the output of Briefcase, so they don't need to be in version control
 * `__pycache__` is a folder created by the Python interpreter, caching compiled bytecode
 * `helloworld.dist-info` is an artefact created when the app module is installed in your development environment
